### PR TITLE
Store first object

### PR DESCRIPTION
**Pull request by karachid**
Adding file storage to JSON

### CODE REVIEW

File: models/base_model.py [hunk 1]
```diff
@@ -29,7 +30,10 @@ def __init__(self, *args, **kwargs):
             self.id = uuid.uuid4().hex
             self.created_at = datetime.now()
             self.updated_at = datetime.now()
-
+            k = f"{self.__class__.__name__}.{self.id}"
```
Comment (Anasoud1) [inline models/base_model.py]: you can directly call new() method

Comment (karachid) [reply models/base_model.py]: Yes you're right, would be better to call new instead of rewriting its logic in constructor

Comment (Anasoud1) [inline models/base_model.py]:
```suggestion
            models.storage.new(self)
```

File: models/base_model.py [hunk 2]
```diff
@@ -29,7 +30,10 @@ def __init__(self, *args, **kwargs):
...
+            if k not in models.storage.all():
```
Comment (Anasoud1) [inline models/base_model.py]:
```suggestion
            
```

Comment (Anasoud1) [reply models/base_model.py]: this line should be removed

Comment (karachid) [reply models/base_model.py]: Changes made and pull request should be merged into main branch

**Pull request merged**